x-pack/legacy/plugins/apm/e2e/cypress.json

```diff
@@ -1,7 +1,7 @@
 {
   "nodeVersion": "system",
   "baseUrl": "http://localhost:5701",
-  "video": false,
+  "video": true,
```

Comment:
In the [video](https://apm-ci.elastic.co/job/apm-ui/job/apm-ui-e2e-tests-mbp/job/PR-61803/59/artifact/cypress/videos/apm.feature.mp4) I see that the login page is not shown but the link directly to the `apm/apm#/services?rangeFrom...`, is that the expected behaviour? or should it click on the login page firstly and the go to apm/apm....?